.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - run:
           name: Ensure we have a clean docker environment
           command: |
-            docker images "fundocker/openshift*"
```

Comment:
This statement was always true so I tried to fix it.

Reply:
Yes, it was written only for logging/debug purpose. Testing it is a good idea! I would prefer having both for logging & test purpose. WDYT?

Reply:
Good idea yes!